cython_setuptools/vendor.py

```diff
@@ -28,7 +28,7 @@ def setup(original_setup_file: str, cythonize: bool = True, **kwargs):
     Arguments:
         original_setup_file (str): it's the "__file__" of the original setup.py
         cythonize (bool): The *cythonize* argument controls the default mode of operation:
-                          set it to ``True`` if you don't distribute C files with your 
+                          set it to ``True`` if you don't distribute C files with your
```

Comment:
Please fix this extra space in original commit in this PR.

Reply:
Fixed, Thanks